usmqe/web/tendrl/mainpage/hosts/models.py

```diff
@@ -0,0 +1,87 @@
+"""
+Common page model for hosts.
+"""
+
+
+from webstr.core import WebstrModel, By, PageElement
+from webstr.common.form import models as form
+import webstr.patternfly.contentviews.models as contentviews
+
+LOCATION = "#/node"
+
+
+class HostsMenuModel(WebstrModel):
+    """
+    Hosts page top menu
+    """
+    header = PageElement(by=By.XPATH, locator="//h1[text()='Hosts']")
+    search = form.TextInput(
+        by=By.XPATH,
+        locator="//*[@placeholder='Search'][@ng-model='node.search']")
+
+
+class HostItemModel(contentviews.ListViewRowModel):
+    """
+    An item (row) in a Hosts list.
+    """
+#    type_label = PageElement(by=By.XPATH, locator=".//span[@ng-bind='host.cluster_type']")
+#    status_icon = PageElement(by=By.XPATH, locator=".//span[contains(@class,'status-icon')]")
+    name_label = PageElement(
+        by=By.XPATH,
+        locator=".//p[contains(@ng-bind,'node.fqdn')]")
+#    ip_label = PageElement(
```

Comment:
The same as for cluster list, I created the framework a few days ago, when it's not implemented yet. Anyway it's just first shot, hence it doesn't support everything.
I'll put here links for https://github.com/Tendrl/specifications/pull/95 and https://github.com/Tendrl/specifications/pull/77